packages/tools/tests/test/visualization/config.json

```diff
@@ -576,6 +576,18 @@
             "playgroundId": "#P7N8YW#1",
             "referenceImage": "highlights.png"
         },
+        {
+            "title": "Area Lights - Standard Material",
+            "renderCount": 500,

```

Comment:
We should not be using `renderCount`. The playground should be updated to only be "done" when the scene is ready. If this is not possible, then we need to fix the Babylon.js code.